project.clj

```diff
@@ -1,11 +1,16 @@
-(defproject clj-cron-parse "0.1.5-SNAPSHOT"
+(defproject amperity/clj-cron-parse "0.1.6-SNAPSHOT"
```

Comment:
doesn't make sense to change version or project name when requesting a PR to this repo. Also, do you know how to transfer ownership of a project in clojars? Since I do not maintain this repo any more (I don't use Clojure these days), I would be more than happy to transfer the project over to you.